src/_data/team.json

```diff
@@ -1,19 +1,23 @@
 [
   {
     "name": "Ola Gasidlo-Brändel",
-    "role": "Project Lead",
+    "role": "Creator & Active Founder",
     "imageUrl": "/assets/images/ola.jpg"
   },
 
   {
-    "name": "Raffaele Pizzari",
-    "role": "Lead Product Engineer",
-    "imageUrl": "/assets/images/raffaele.jpg"
```

Comment:
Please move Rafa's entry and image to the specialthanks.json instead! It's where we list Alumni for their work on Deven 💛 

(Shown on the Contributors Page)

Reply:
good point will change that